src/storage-preview/azext_storage_preview/_params.py

```diff
@@ -27,6 +28,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     t_base_blob_service = self.get_sdk('blob.baseblobservice#BaseBlobService')
     t_file_service = self.get_sdk('file#FileService')
     t_table_service = get_table_data_type(self.cli_ctx, 'table', 'TableService')
+    t_queue_service = self.get_sdk('_queue_service_client#QueueServiceClient')
```

Comment:
You have to specify resource type here when you use track 2 SDK because previous default one is for track1 and track1 services don't need to specify it here. Please check if completer works.

Reply:
done